sormas-backend/src/main/java/de/symeda/sormas/backend/sample/SampleService.java

```diff
@@ -1094,7 +1094,8 @@ public Predicate createDefaultFilter(CriteriaBuilder cb, Root<Sample> root) {
 		return cb.isFalse(root.get(Sample.DELETED));
 	}
 
-	public Boolean isSampleEditAllowed(Sample sample) {
+
+	public Boolean isEditAllowed(Sample sample) {
```

Comment:
This should not be an Object Boolean, and the current implementation does not support it. 
`inJurisdictionOrOwned(sample).getInJurisdiction()::Boolean` will run into NullPointerException.

Reply:
fixed